go/config/host_config.go

```diff
@@ -255,5 +261,6 @@ func DefaultHostParsedConfig() *HostInputConfig {
 		RollupInterval:            5 * time.Second,
 		L1BlockTime:               15 * time.Second,
 		IsInboundP2PDisabled:      false,
+		MaxRollupSize:             1024 * 64,
```

Comment:
Similarly, how does this max size compare with whatever the enclave-side rollup limiter is capped at?

Maybe they could use the same config, because people might increase this thinking it will help but if the limiter is more stingy then it won't.

Reply:
Though SHA-256 nominally offers a 256-bit output, no weakness about it is known when the output is truncated to 128 bits, except, of course, weaknesses inherent to the shorter output length; e.g. collision resistance drops from the infeasible 2128 to the possible (but hard) 264.

And it sounds like keccak is similarly secure for truncation. So I guess it's all good?

Reply:
This flag should be the same in both host and enclave.